packages/jest-create-cache-key-function/src/index.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import {createHash} from 'crypto';
+import {relative} from 'path';
+import {readFileSync} from 'graceful-fs';
+import type {Config, ProjectConfig} from '@jest/types';
+
+type CacheKeyOptions = {
+  config: ProjectConfig;
+  instrument: boolean;
+};
+
+type GetCacheKeyFunction = (
+  fileData: string,
+  filePath: Config.Path,
+  configStr: string,
+  options: CacheKeyOptions,
+) => string;
+
+function getGlobalCacheKey(files: Array<string>, values: Array<string>) {
+  const dependencies = require('../package').dependencies;
+  return [
+    process.env.NODE_ENV,
+    process.env.BABEL_ENV,
+    dependencies && dependencies['babel-preset-fbjs'],
```

Comment:
this will never be true as the local package doesn't depend on this, no? I might be missing something